profiling/src/php_ffi.c

```diff
@@ -75,6 +75,12 @@ void datadog_php_profiling_copy_string_view_into_zval(zval *dest, zai_string_vie
     }
 }
 
+void ddog_php_prof_copy_long_into_zval(zval *dest, long num) {
+    ZEND_ASSERT(dest);
+    ZVAL_LONG(dest, num);
+    return;
+}
```

Comment:
I suppose we could add a `ZEND_ASSERT` to be sure `dest` is undefined or a value which doesn't need refcounts nor dtor'd?